.github/workflows/changesets.yaml

```
@@ -36,3 +35,53 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - name: Tag Dashboard
+        id: dashboard
+        if: steps.changesets.outputs.hasChangesets == 'false'
+        run: |
+          package="@nhost/dashboard"
+          version=$(jq -r .version dashboard/package.json)
+          tag="$package@$version"
+          git tag $tag 2>/dev/null && (git push origin --tags ; echo "dashboardVersion=$tag" >> $GITHUB_OUTPUT) || true
+
+  publish:
+    runs-on: ubuntu-latest
+    needs: version
+    if: needs.version.outputs.hasChangesets == 'false' && needs.version.outputs.dashboardVersion != ''
```

`hasChangesets` is always`'false'` when there is a dashboard version
```suggestion
    if: needs.version.outputs.dashboardVersion != ''
```